packages/eos-components/src/components/TopicCard.vue

```diff
@@ -0,0 +1,85 @@
+<template>
+  <b-card
+    noBody
+    class="my-2 rounded-lg"
+    :class="{
+      shadow: !isHovered,
+      'shadow-lg': isHovered,
+    }"
+  >
+    <ContentLink :url="url" @isHovered="(hovered) => isHovered = hovered">
+      <b-card-body>
+        <b-card-title>
+          <VClamp
+            autoresize
+            :maxLines="3"
+          >
+            {{ node.title }}
+          </VClamp>
+        </b-card-title>
+        <b-card-text>
+          <VClamp
+            autoresize
+            :maxLines="5"
+          >
+            {{ node.description }}
+          </VClamp>
+        </b-card-text>
+        <b-card-sub-title class="text-truncate">
+          {{ subtitle }}
+        </b-card-sub-title>
+      </b-card-body>
+    </ContentLink>
+  </b-card>
+</template>
+
+<script>
+import VClamp from 'vue-clamp';
+
+export default {
+  name: 'TopicCard',
+  components: {
+    VClamp,
+  },
+  props: {
+    node: Object,
+    subtitle: String,
+    url: String,
+  },
+  data() {
+    return {
+      isHovered: false,
+    };
+  },
+};
+</script>
+
+<style lang="scss" scoped>
+@import '../styles.scss';
+
+.card {
+  background-color: $primary;
+  transition: all ease .4s;
+}
+
+.card-body {
+  // This 8 is an estimation:
+  min-height: card-body-height(8);
```

Comment:
The card image height is calculated with a percentage and a padding top: https://github.com/endlessm/kolibri-explore-plugin/blob/master/packages/template-ui/src/components/RegularCard.vue#L79